shared/checks/oval/installed_OS_is_uosv20.xml

```diff
@@ -0,0 +1,28 @@
+<def-group>
+  <definition class="inventory" id="installed_OS_is_uosv20" version="1">
+    <metadata>
+      <title>Uniontech OS V20</title>
+      <affected family="unix">
+        <platform>multi_platform_all</platform>
+      </affected>
+      <reference ref_id="cpe:/o:Uniontech OS:v20" source="CPE" />
+      <description>The operating system installed on the system is Uniontech OS</description>
+    </metadata>
+    <criteria operator="AND">
+      <extend_definition comment="Installed OS is part of the Unix family" definition_ref="installed_OS_is_part_of_Unix_family" />
+      <criterion comment="Uniontech OS V20 is installed" test_ref="test_uosv20" />
+    </criteria>
+  </definition>
+
+  <linux:rpminfo_test check="all" check_existence="at_least_one_exists" comment="uos-release is version v20" id="test_uosv20" version="1">
+    <linux:object object_ref="obj_uosv20" />
+    <linux:state state_ref="state_uosv20" />
+  </linux:rpminfo_test>
+  <linux:rpminfo_state id="state_uosv20" version="1">
+    <linux:version operation="pattern match">^v20.*$</linux:version>
+  </linux:rpminfo_state>
+  <linux:rpminfo_object id="obj_uosv20" version="1">
+    <linux:name>alinux-release</linux:name>
```

Comment:
Can you please confirm this?

Reply:
maybe if it's based on `alinux-release`, this product can be turned into a derivative product of the upcoming `alinux` product

Reply:
Sorry, this should be UOS release. UOS and alinux are two different products, it  have been omitted. I will correct them